docs/source/quantization.rst

```diff
@@ -186,7 +188,7 @@ Layers for the quantization-aware training
 Observers for computing the quantization parameters
 
 * :class:`~torch.quantization.MinMaxObserver` — Derives the quantization parameters from the running minimum and maximum of the observed tensor inputs (per tensor variant)
-* :class:`~torch.quantization.MovingAverageObserver` — Derives the quantization parameters from the running averages of the minimums and maximums of the observed tensor inputs (per tensor variant)
+* ``MovingAverageObserver`` — Derives the quantization parameters from the running averages of the minimums and maximums of the observed tensor inputs (per tensor variant)
```

Comment:
I think it will work if you change it to ```:class:`~torch.quantization.MovingAverageMinMaxObserver` ```

Reply:
That works, thank you for the pointer